pycbc/inference/sampler/__init__.py

```diff
@@ -101,4 +101,10 @@ def load_from_config(cp, model, **kwargs):
         return DummySampler.from_config(cp, model, **kwargs)
 
     name = cp.get('sampler', 'name')
-    return samplers[name].from_config(cp, model, **kwargs)
+    try:
+        return samplers[name].from_config(cp, model, **kwargs)
+    except KeyError:
+        raise ImportError(f"No accessible sampler named {name}. Please check"
+                          f" if the name is correct or the required package"
+                          f" for this sampler is installed correctly."
+                          f" Accessible samplers: {list(samplers.keys())}")
```

Comment:
Thanks for looking at this!

Super minor suggestions:
 - I've removed the fstrings which don't use formatting
 - moved the spaces within the strings to the end of the lines rather than the start
 - list formatting in fstrings isn't that nice, so I've changed it to comma-separated
 - Minor wording change from 'accessible' to 'available'

```suggestion
        raise ImportError(
            f"No available sampler named {name}. Please check "
            "if the name is correct or the required package "
            "for this sampler is installed correctly. "
            f"Available samplers: {', '.join(list(samplers.keys()))}"
        )
```